src/webgl/p5.RenderBuffer.js

```diff
@@ -27,45 +49,77 @@ p5.RenderBuffer = class {
       model = geometry;
     }
 
+    let geometryData = geometry;
+    if (this._namespace) {
```

Comment:
Is there some redundancy of if statements and loops here? Any reason to not just combine into one `if` and one `for`